static/src/stylesheets/module/facia-garnett/_item.scss

```diff
@@ -555,9 +560,15 @@ $block-height: 58px;
         background: linear-gradient(to bottom, rgba(0, 0, 0, 0) 0%,  rgba(0, 0, 0, .8) 100%);
         color: $brightness-100;
         padding: 60px 8px 8px;
+    }
+}
 
-        @include mq($until: tablet) {
-            @include font-size(12, 16);
+.fc-item--has-floating-sublinks .fc-item__captioned-image {
+    figcaption {
+        @include mq(tablet) {
+            background: linear-gradient(to top, rgba(0, 0, 0, 0) 0%,  rgba(0, 0, 0, .8) 100%);
+            padding: 8px 8px 60px;
+            bottom: inherit;
```

Comment:
O would be tempted to keep this in the main SASS block:

```sass
figcaption {
    /* … */

    .fc-item--has-floating-sublinks .fc-item__captioned-image & {
        /* … */
    }
}
```

Reply:
https://github.com/guardian/frontend/pull/24860/commits/47e40d6741d46c9f0628ead3050836e0e85e9cf8 is probably the best I can do, the & operator gets a bit confused if I put it in a figcaption

Reply:
This was exactly what I meant! Keeping things more compact is good.